iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeDescriptor.java

```diff
@@ -168,11 +168,16 @@ private void loadProperties(Properties properties) throws BadNodeUrlException, I
                     IoTDBConstant.CN_CONSENSUS_PORT, String.valueOf(conf.getConsensusPort()))
                 .trim()));
 
-    // TODO: Enable multiple target_config_node_list
-    String targetConfigNodes =
-        properties.getProperty(IoTDBConstant.CN_TARGET_CONFIG_NODE_LIST, null);
-    if (targetConfigNodes != null) {
-      conf.setTargetConfigNode(NodeUrlUtils.parseTEndPointUrl(targetConfigNodes.trim()));
+    String seedConfigNode = properties.getProperty(IoTDBConstant.CN_SEED_CONFIG_NODE, null);
+    if (seedConfigNode == null) {
+      seedConfigNode = properties.getProperty(IoTDBConstant.CN_TARGET_CONFIG_NODE_LIST, null);
+      LOGGER.warn(
+          "The parameter cn_target_config_node_list has been abandoned, "
+              + "only the first ConfigNode address will be used to join in the cluster. "
+              + "Please use cn_seed_config_node instead.");
+    }
+    if (seedConfigNode != null) {
+      conf.setSeedConfigNode(NodeUrlUtils.parseTEndPointUrls(seedConfigNode.trim()).get(0));
```

Comment:
move to line 177?

Reply:
These lines should be here to keep compability.

Reply:
Fixed